src/templates/index.html

```diff
@@ -26,17 +26,24 @@ <h3 class="greeting-content">Quote of the day</h3>
 <script>
     function heartClicked(button) {
         var icon = button.querySelector('i');
+        
+        // Toggle the heart icon appearance
         icon.classList.toggle('far');
         icon.classList.toggle('fas');
-
-        // If the quote is hearted
-        if (icon.classList.contains('fas')) {
-            // Send a request to the server to save the vote
-            fetch('/?handler=HeartQuote', {
+        // Determine if it's a like or unlike action
+        const isNowLiked = icon.classList.contains('fas');
+        const action = isNowLiked ? 'like' : 'unlike';
```

Comment:
We don't need the "action". We will only send the request if the quote is liked. We can use the old code with just the endpoint update.